packages/react/src/utils/css-in-js.ts

```diff
@@ -0,0 +1,50 @@
+import {
+  emotionCoreVersion,
+  emotionStyledVersion,
+  styledComponentVersion
+} from './versions';
+import { PackageDependencies } from './dependencies';
+
+export function reduceCssInJsModules(
+  extraPackageDependencies: null | PackageDependencies,
+  style: string
+): null | PackageDependencies {
+  const deps = extraPackageDependencies
+    ? extraPackageDependencies.dependencies
+    : {};
+  const devDeps = extraPackageDependencies
+    ? extraPackageDependencies.devDependencies
+    : {};
+  switch (style) {
```

Comment:
Perhaps this can just be an object of things
```ts
{
  'styled-components': {
    dependencies: {
      'styled-components': styledComponentVersion
    },
    devDependencies: {}
  },
  emotion: {
    ...
  }
}
```